modules/transformations/redshift/lib/center_lib/center_median.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021, CARTO
```

Comment:
As I commented in processing, maybe it makes sense to introduce some unit-tests as this code is not just taken from somewhere else.